go-jankari-bot/README.md

```diff
@@ -0,0 +1,74 @@
+### Jankaritech's rocket.chat bot
```

Comment:
```suggestion
### teamup rocket.chat bot
```

also the name of the folder should be something that we can recognize also in 2 month time